src/js/user-profile/helpers.jsx

```diff
@@ -1,18 +1,28 @@
 import Raven from 'raven-js';
 
-export const formTitles = {
-  '21P-527EZ': 'Veterans pension benefits (21P-527EZ)',
-  '21P-530': 'burial benefits (21P-530)',
-  '1010ez': 'health care (10-10EZ)',
-  '22-1990': 'education benefits (22-1990)',
-  '22-1990E': 'education benefits (22-1990E)',
-  '22-1990N': 'education benefits (22-1990N)',
-  '22-1995': 'education benefits (22-1995)',
-  '22-5490': 'education benefits (22-5490)',
-  '22-5495': 'education benefits (22-5495)',
+export const formBenefits = {
+  '21P-527EZ': 'Veterans pension benefits',
+  '21P-530': 'burial benefits',
+  '1010ez': 'health care',
+  '22-1990': 'education benefits',
+  '22-1990E': 'education benefits',
+  '22-1990N': 'education benefits',
+  '22-1995': 'education benefits',
+  '22-5490': 'education benefits',
+  '22-5495': 'education benefits',
   '40-10007': 'pre-need determination of eligibility in a VA National Cemetery'
 };
 
+export const formTitles = Object.keys(formBenefits).reduce((titles, key) => {
+  titles[key] = key === '40-10007' // eslint-disable-line no-nested-ternary no-param-reassign  
+    ? formBenefits[key]
+    : key === '1010ez'
+      ? `${formBenefits[key]} (10-10EZ)`
```

Comment:
Instead of this extra conditional, you could always upper case the key: ``${formBenefits[key]} (${key.toUpperCase()})``

Reply:
Thanks for the recommendation @jbalboni. It looks like a dash would need to be inserted into the formID, is that an option?